src/main/java/Problem1/LevelOrderTraverse.java

```diff
@@ -2,11 +2,47 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Queue;
+import java.util.Stack;
+import java.util.concurrent.ArrayBlockingQueue;
 
 public class LevelOrderTraverse {
-    public static List<List<Integer>> levelOrder(TreeNode<Integer> root) {
-        // homework
+    public static List<List<Integer>> levelOrder(TreeNode<Integer> root)
+    {
         List<List<Integer>> result = new ArrayList<>();
-        return result;  // place holder
+        Queue<TreeNode> prevLayer = new ArrayBlockingQueue<TreeNode>(20);
```

Comment:
based on the discussion in the youtube video, this 20 definitely deserves an explanation, no?